internal/pkg/deploy/cloudformation/workload.go

```diff
@@ -39,3 +39,8 @@ func (cf CloudFormation) DeployService(conf StackConfiguration, opts ...cloudfor
 func (cf CloudFormation) DeleteService(in deploy.DeleteWorkloadInput) error {
 	return cf.cfnClient.DeleteAndWait(fmt.Sprintf("%s-%s-%s", in.AppName, in.EnvName, in.Name))
 }
+
+// DeleteJob removes the CloudFormation stack of a deployed job.
+func (cf CloudFormation) DeleteJob(in deploy.DeleteJobInput) error {
```

Comment:
maybe we could let this method take `deploy.DeleteWorkloadInput` as input as well and wraps the error returned from `cf.cfnClient.DeleteAndWait(fmt.Sprintf("%s-%s-%s", in.AppName, in.EnvName, in.Name))
}` to be "job specific", so that the error message could be more intuitive (since `DeleteAndWait` is a public method)

Reply:
I believe that error is already being wrapped where `DeleteJob` and `DeleteService` are being called (within `deleteStacks` in `job_delete.go` and `svc_delete.go`, respectively. I did change both to take `deploy.DeleteWorkloadInput` though!

Reply:
yeah but without wrapping the error, it would be as `delete stack ${stackName}: some error`, or `wait until stack ${stackName} delete is complete: some error`, which are not very intuitive because stack info is actually our implementation details.

However, if we wrap the error it will be `delete service ${serviceName}: delete stack ${stackName}: some error`, which might be better? What do you think?

Reply:
Yes! Sorry, wrapped now!